src/spaces/productspace.jl

```diff
@@ -31,6 +31,8 @@ Base.axes(P::ProductSpace, n::Int) = axes(P.spaces[n])
 
 dual(P::ProductSpace{<:ElementarySpace,0}) = P
 dual(P::ProductSpace) = ProductSpace(map(dual, reverse(P.spaces)))
+flip(P::ProductSpace{<:ElementarySpace,0}) = P
+flip(P::ProductSpace) = ProductSpace(map(flip, reverse(P.spaces)))
```

Comment:
Do we want reverse in this case. Not sure where this is coming up?

Reply:
I apparently got confused because of the MATLAB syntax for flip. In BraidingTensor, there is a check somewhere that is meant to verify if the input homspace is `V1 ⊗ V2 ← V2 ⊗ V1`, and somehow I used `flip(codomain(W)) == domain(W)` for this. I changed it to use `Base.reverse(ProductSpace)`, which should be fine?

Reply:
Yes I think the current version is better. This is the doc string for flip:

```
Return a single vector space of type `S` that has the same value of [`isdual`](@ref) as
`dual(V)`, but yet is isomorphic to `V` rather than to `dual(V)`.
```